cloud/kubernetes/helm/yugabyte/templates/service.yaml

```diff
@@ -23,27 +23,34 @@ spec:
   selector:
     app: "{{ .label }}"
 
-{{- if .hasLoadBalancer }}
 ---
+{{- $service := . -}}
+{{- range $type, $metadata :=  $root.Values.serviceEndpoints }}
+{{- if eq $service.label (printf "yb-%s" $type) }}
+{{- with $metadata }}
 apiVersion: v1
 kind: Service
 metadata:
-  name: "{{ .label }}-ui"
+  name: "{{ $service.label }}-service"
   labels:
-    app: "{{ .label }}"
+    app: "{{ $service.label }}"
     heritage: {{ $root.Release.Service | quote }}
     release: {{ $root.Release.Name | quote }}
     chart: "{{ $root.Chart.Name }}-{{ $root.Chart.Version }}"
     component: "{{ $root.Values.Component }}"
 spec:
   clusterIP:
   ports:
-    - name: ui
-      port: 7000
+    {{- range $label, $port :=  $metadata.ports }}
+    - name: {{ $label | quote }}
+      port: {{ $port }}
+    {{- end}}
   selector:
-    app: "{{ .label }}"
-  type: LoadBalancer
-{{- end }}
+    app: "{{ $service.label }}"
```

Comment:
By selecting this app, does the lbs get deployed into the correct namespace? Was not sure where we set the namespace for this lbs.